features/customers.feature

```diff
@@ -159,4 +159,107 @@ Scenario: Create a Customer with Missing Password
     And I should see "Required field" in the "Password" error string
     When I set the "Password" to "test"
     And I press the "Create" button
+    Then The "Password" error string should be gone
+
+Scenario: Update a Customer
+    When I visit the "Home Page"
+    And I press the "Search" button
+    Then I should see the message "Success"
+    When I copy the "id" field
+    And I set the "First Name" to "Test FN"
+    And I set the "Last Name" to "Test LN"
+    And I set the "Email" to "test@gmail.com"
+    And I set the "Password" to "test788pass"
+    And I select "True" in the "Active" dropdown
+    And I set the "street" to "10th Casselberry Way"
```

Comment:
~street~ Street
Similarly, Can you make all of the fields and button names to be PascalCase with first letter capitalized?

Reply:
fixed